app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
remove this and add new strings into all files please

Reply:
Done